dubbo-remoting/dubbo-remoting-netty4/src/test/java/org/apache/dubbo/remoting/transport/netty4/ConnectionTest.java

```diff
@@ -138,6 +138,7 @@ void connectSyncTest() throws Throwable {
 
         nettyPortUnificationServer.bind();
         // auto reconnect
+        Thread.sleep(60000);
```

Comment:
Use awaitility to verify

Reply:
I've made the adjustments that use awaitility.